array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,12 @@
+/* exported drop */
+function drop(array, count) {
+  var dropArray = [];
+  for (var i = count; i < array.length; i++) {
+    if (array[count]) {
+      dropArray.push(array[i]);
+    } else {
+      break;
+    }
```

Comment:
You should not need an `if/else` for this. Try to simplify your solution. Think carefully about your initialization and condition expressions in the `for` loop.